service/src/test/java/cz/muni/fi/pa165/BottleTypeServiceTest.java

```diff
@@ -0,0 +1,167 @@
+package cz.muni.fi.pa165;
+
+import cz.muni.fi.pa165.config.ServiceConfiguration;
+import cz.muni.fi.pa165.dao.BottleTypeDAO;
+import cz.muni.fi.pa165.entity.BottleType;
+import cz.muni.fi.pa165.enums.AlcoholType;
+import cz.muni.fi.pa165.service.BottleTypeService;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Martin Sumera
+ */
+@ContextConfiguration(classes = ServiceConfiguration.class)
+public class BottleTypeServiceTest extends AbstractServiceTest {
+
+    @Mock
+    private BottleTypeDAO bottleTypeDAO;
+
+    @InjectMocks
+    @Autowired
+    private BottleTypeService bottleTypeService;
+
+    private BottleType firstBottleType;
+
+    private BottleType secondBottleType;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+
+        firstBottleType = new BottleType();
+        firstBottleType.setName("name1");
+        firstBottleType.setType(AlcoholType.RUM);
+
+        secondBottleType = new BottleType();
+        secondBottleType.setName("name2");
+        secondBottleType.setType(AlcoholType.VODKA);
+    }
+
+    @Test
+    public void findAll_withEmptyList_returnsEmptyList() {
```

Comment:
Tieto nazvy testov sa mi moc nepacia, vsade ine pouziva CamelCase, asi by som to prerobil